__fixtures__/example-todo-main/api/src/lib/polyfill.js

```diff
@@ -1 +1,288 @@
-''.replaceAll('a', 'b')
\ No newline at end of file
+/**
+ * Redwood targets Node.js 12, but that doesn't factor into what gets polyfilled
+ * because Redwood uses the plugin-transform-runtime polyfill strategy.
+ *
+ * Also, plugin-transform-runtime is pinned to core-js v3.0.0,
+ * so the list of available polyfill is a little outdated.
+ * Some "ES Next" polyfills have landed in v12+ Node.js versions.
+ *
+ * Key:
```

Comment:
These comments are super helpful Dom - but its not really visible when you open the build_api test.

I wonder if its possible to group some of the tests into "it" or "test" blocks to help get the point across? I think some of the nuance you explained to me the other day got lost in the code.

Just a suggestion

Reply:
Working on better testing structure:

<img width="499" alt="image" src="https://user-images.githubusercontent.com/32992335/185088849-e7fda08f-3e9c-4938-9c21-49319a4cd8ba.png">